pkg/cli/helm/contourclient.go

```diff
@@ -120,7 +120,7 @@ func AddContourValues(helmChart *chart.Chart, options ContourOptions) error {
 		}
 
 		servicePortsNode := serviceNode["ports"].(map[string]any)
-		if serviceNode == nil {
```

Comment:
Was this a bug? looks like it might have been. This is why cleanup like this is helpful 👍